modules/cco-ccoctl-deleting-sts-resources.adoc

```diff
@@ -2,52 +2,82 @@
+:cp: GCP
+:cp-name: gcp
+:gcp-workload-id:
+endif::[]
+ifeval::["{context}" == "uninstall-cluster-azure"]
+:cp-first: Microsoft Azure
+:cp: Azure
+:cp-name: azure
+:azure-workload-id:
 endif::[]
 
 :_content-type: PROCEDURE
 [id="cco-ccoctl-deleting-sts-resources_{context}"]
-ifdef::aws-sts[]
-= Deleting AWS resources with the Cloud Credential Operator utility
-
-To clean up resources after uninstalling an {product-title} cluster with the Cloud Credential Operator (CCO) in manual mode with STS, you can use the CCO utility (`ccoctl`) to remove the AWS resources that `ccoctl` created during installation.
-endif::aws-sts[]
-
-ifdef::google-cloud-platform[]
-= Deleting GCP resources with the Cloud Credential Operator utility
+= Deleting {cp-first} resources with the Cloud Credential Operator utility
 
-To clean up resources after uninstalling an {product-title} cluster with the Cloud Credential Operator (CCO) in manual mode with GCP Workload Identity, you can use the CCO utility (`ccoctl`) to remove the GCP resources that `ccoctl` created during installation.
-endif::google-cloud-platform[]
+After uninstalling an {product-title} cluster that uses short-term credentials managed outside the cluster, you can use the CCO utility (`ccoctl`) to remove the {cp-first} ({cp}) resources that `ccoctl` created during installation.
 
 .Prerequisites
 
 * Extract and prepare the `ccoctl` binary.
-ifdef::aws-sts[]
-* Install an {product-title} cluster with the CCO in manual mode with STS.
-endif::aws-sts[]
-ifdef::google-cloud-platform[]
-* Install an {product-title} cluster with the CCO in manual mode with GCP Workload Identity.
-endif::google-cloud-platform[]
+* Uninstall a {cp} cluster that uses short-term credentials.
 
 .Procedure
-
-ifdef::aws-sts[]
-* Delete the AWS resources that `ccoctl` created:
+//GCP has extra prep steps
+ifdef::gcp-workload-id[]
+. Set a `$RELEASE_IMAGE` variable with the release image from your installation file by running the following command:
 +
 [source,terminal]
 ----
-$ ccoctl aws delete \
-  --name=<name> \ <1>
-  --region=<aws_region> <2>
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
+----
+
+. Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm release extract \
+  --credentials-requests \
+  --cloud={cp-name} \
+  --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <1>
+  --from=$RELEASE_IMAGE
+----
++
+<1> `credrequests` is the directory where the list of `CredentialsRequest` objects is stored. This command creates the directory if it does not exist.
+
+. Delete the {cp} resources that `ccoctl` created by running the following command:
+endif::gcp-workload-id[]
+ifdef::aws-sts,azure-workload-id[]
+* Delete the {cp} resources that `ccoctl` created by running the following command:
+endif::aws-sts,azure-workload-id[]
```

Comment:
Azure requires a couple other parameters, see note here
https://github.com/openshift/openshift-docs/pull/62148#issuecomment-1761055077

Reply:
Added in https://github.com/openshift/openshift-docs/pull/64001/commits/8b47f7af10829104ac6569615d52c280dbcd4240